### ISSUE TITLE
fontUnit does not work

### ISSUE THREAD

**Issue by pikkupullo** [sollenne/angular-fittext#26]
Tried to use this, and noticed that fontUnit does not work. When adding [fontUnit], the resizing gets broken. If only using for example [maxFontSize] without [fontUnit] it works with the default value (px). Would like to use vh with this one. Otherwise nice work!

**Comment by sollenne** [comment]
Is this still an issue @pikkupullo ? I'm doing some work to it. Have you thought about creating a PR?

**Comment by markolyden** [comment]
@sollenne Yes, a minor problem but still exists. I have a bootstrap carousel in mobile in which might contain long descriptions, and those go out of screen without resizing. Breaking those into multiple lines is not really an option.

Haven't thought about putting my fingers in this one currently :D Might do it if required, but currently way too many things to do, so it might take a while.

Thanks!
